plugins/sbt/graphql-java-codegen-sbt-plugin/src/main/scala/io/github/dreamylost/graphql/codegen/GraphQLCodegenPlugin.scala

```diff
@@ -114,6 +114,7 @@ class GraphQLCodegenPlugin(configuration: Configuration, private[codegen] val co
     generateToString := MappingConfigConstants.DEFAULT_TO_STRING,
     // parent interfaces configs:
     parentInterfaces := parentInterfacesConfig,
+    generateAllMethodInProjection := MappingConfigConstants.DEFAULT_GENERATE_ALL_METHOD,
```

Comment:
This key must declare in `GraphQLCodegenKeys.scala `

Reply:
e.g: `val generateAllMethodInProjection = settingKey[Boolean](you description for this option)`